packages/e2e-tests/specs/editor/various/preferences.test.js

```diff
@@ -29,8 +29,10 @@ describe( 'preferences', () => {
 	}
 
 	it( 'remembers sidebar dismissal between sessions', async () => {
+		const documentLabel = await getDocumentLabel();
+
 		// Open by default.
-		expect( await getActiveSidebarTabText() ).toBe( 'Document' );
+		expect( await getActiveSidebarTabText() ).toBe( documentLabel );
```

Comment:
If possible, I would always keep the values in these tests explicit. We already know that we are dealing with Posts or Pages, so there is great benefit in reducing the complexity of tests:

```suggestion
		expect( await getActiveSidebarTabText() ).toBe( 'Post' );
```